backon/src/lib.rs

```diff
@@ -31,16 +31,53 @@
 //!
 //! # Sleep
 //!
-//! Retry in BackON requires an implementation for sleeping. BackON will accept a [`Sleeper`] to pause for a specified duration.
+//! Retry in BackON requires an implementation for sleeping, such an implementation
+//! is called a Sleeper, it will implement [`Sleeper`] or [`BlockingSleeper`] depending
```

Comment:
> such an implementation is called a Sleeper, it will implement [`Sleeper`] or [`BlockingSleeper`] depending on if it is going to be used in an asynchronous context.

I try to differentiate "Sleeper" from "\`Sleeper\`" (quoted with backticks), the former is a general concept, which can be a "\`Sleeper\`" or "\`BlockingSleeper`\", while the later is just the async Sleeper trait.

Reply:
Makes sense.